dbschema/src/main/java/org/folio/rest/persist/ddlgen/Index.java

```diff
@@ -83,53 +85,40 @@ public String getFinalSqlExpression(String tableLoc) {
         if(i != 0) {
           result .append(" , ");
         }
-        appendExpandedTerm(tableLoc, splitIndex[i], result);
+        result.append(formatExpandedTerm(tableLoc, splitIndex[i]));
       }
       result.append(")");
       return result.toString();
     }
 
   }
-  public static void appendExpandedTerm(String tableLoc, String  currentTerm, StringBuilder result) {
-    String [] rawExpandedTerm = currentTerm.split("\\.");
-    StringBuilder expandedTerm = formatExpandedTerm(tableLoc + ".jsonb",rawExpandedTerm);
-    result.append(expandedTerm);
-  }
 
-  public static StringBuilder formatExpandedTerm(String table, String[] rawExpandedTerm) {
+  public static StringBuilder formatExpandedTerm(String table, String term) {
```

Comment:
Can you change this to `static void appendExpandedTerm(String table, String term, StringBuilder result)`?
Is there a need to have this method `public`?